bindings/pyroot_experimental/PyROOT/src/PyzPythonHelpers.cxx

```diff
@@ -75,14 +74,40 @@ PyObject *PyROOT::GetVectorDataPointer(PyObject * /*self*/, PyObject *args)
    unsigned long long pointer = 0;
    std::stringstream code;
    code << "*((long*)" << &pointer << ") = reinterpret_cast<long>(reinterpret_cast<" << cppname << "*>(" << cppobj
-        << ")->data())";
+        << ")->" << method << "())";
    gInterpreter->Calc(code.str().c_str());
 
    // Return pointer as integer
    PyObject *pypointer = PyLong_FromUnsignedLongLong(pointer);
    return pypointer;
 }
 
+////////////////////////////////////////////////////////////////////////////
+/// \brief Get pointer to the data of a vector
+/// \param[in] self Always null, since this is a module function.
+/// \param[in] args[0] Data-type of the C++ object as Python string
+/// \param[in] args[1] Python representation of the C++ object.
+///
+/// This function returns the pointer to the data of a vector as an Python
+/// integer.
+PyObject *PyROOT::GetVectorDataPointer(PyObject * /*self*/, PyObject *args)
```

Comment:
In order to reduce the (growing) number of functions of the PyROOT extension module interface, why don't we just have one `GetDataPointer` function in the interface, and we pass (from Python) "data" or "GetData" in each case?

Reply:
I'll go for this, thanks!

Reply:
Done